x/evm/spec/01_concepts.md

```diff
@@ -0,0 +1,111 @@
+<!--
+order: 1
+-->
+
+# Concepts
```

Comment:
After migration is complete we will have to adjust the [`pre.sh` script for docs building](https://github.com/evmos/evmos/blob/main/docs/pre.sh#L25-L46) since we will not need to sparse checkout the Ethermint repo anymore.